### ISSUE TITLE
FATAL ERROR:EOF from RADB (dequeue, ripe) Exit Code 255

### ISSUE THREAD

**Issue by smaxx1337** [snar/bgpq3#52]
Hello,

running this command:
```
bgpq3 -d -h rr.ntt.net -S RIPE,APNIC,AFRINIC,ARIN,NTTCOM,ALTDB,BBOI,BELL,JPIRR,LEVEL3,RADB,RGNET,TC -6 -A -j -r 16 -R 48 -l prefix_list AS-PCH
```
Gives me following error:
**FATAL ERROR:EOF from RADB (dequeue, ripe)**

So I tried it with the -d parameter and here are few lines before it seems to crash:
```
DEBUG: bgpq_expander.c:250 bgpq_pipeline_dequeue_ripe dequeuer(ripe): got route6 2001:1398:274::/48
DEBUG: bgpq_expander.c:250 bgpq_pipeline_dequeue_ripe dequeuer(ripe): got route6 2001:1398:121::/48
DEBUG: bgpq_expander.c:250 bgpq_pipeline_dequeue_ripe dequeuer(ripe): got route6 2001:1398:275::/48
DEBUG: bgpq_expander.c:250 bgpq_pipeline_dequeue_ripe dequeuer(ripe): got route6 2405:3780::/32
DEBUG: bgpq_expander.c:250 bgpq_pipeline_dequeue_ripe dequeuer(ripe): got route6 2620:95:8000::/48
DEBUG: bgpq_expander.c:250 bgpq_pipeline_dequeue_ripe dequeuer(ripe): got route6 2620:95:8000::/48
DEBUG: bgpq_expander.c:250 bgpq_pipeline_dequeue_ripe dequeuer(ripe): got route6 2a0d:e8c0::/29
FATAL ERROR:EOF from RADB (dequeue, ripe)
```

It starts with:
```
DEBUG: bgpq_expander.c:633 bgpq_expand Acquired sendbuf of 4608 bytes
DEBUG: bgpq_expander.c:524 bgpq_expand_radb expander: sending '!iAS-PCH,1
'
```
Then, i get this:
```
DEBUG: bgpq_expander.c:394 bgpq_pipeline expander: sending '-T route6 -i origin as27
' (queued 0 of 4608)
```
(This goes for few AS numbers)

Aaand then it does the "got route6" thing and then only at this certain line:
`DEBUG: bgpq_expander.c:250 bgpq_pipeline_dequeue_ripe dequeuer(ripe): got route6 2a0d:e8c0::/29` it crashes.

Any idea how to fix this?
 
I'm running BGPQ3 on a CentOS7 machine, installed with "yum install bgpq3", the BGPQ3 version is 0.1.31.

**Comment by smaxx1337** [comment]
I tried setting these things in /etc/sysctl.conf:
```
net.ipv4.tcp_window_scaling=1
net.core.rmem_max=2097152
net.core.wmem_max=2097152
net.ipv4.tcp_rmem="4096 87380 2097152"
net.ipv4.tcp_wmem="4096 65536 2097152"
```
But it didn't change anything, its still stuck. Also this crash happens with every AS.

**Comment by smaxx1337** [comment]
Fixed it by removing the 0.1.31 version and installing the 0.1.35, for anyone that runs into this error:
just git clone this repository, ensure that you have the build-essentials (or just yum install "Development Tools" on CentOS7) installed and compile it with:
./configure && make && make install - then move the bgpq3 binary in /usr/bin/bgpq3.
